core/src/main/scala/dagr/core/execsystem/TaskExecutionRunner.scala

```diff
@@ -89,7 +89,7 @@ private object TaskRunner {
   }
 
   /** Simple class that sets the exit code to zero */
-  class NoOpTaskRunner(task: UnitTask) extends TaskRunnable(task = task) {
+  class NoOpTaskExecutionRunner(task: UnitTask) extends TaskRunnable(task = task) {
```

Comment:
Maybe rename this to `SimulationTaskRunner` or comment that it's used for simulation.  I had to go hunting to figure out why you'd want this.